### PR TITLE
[Reproduce tool] Bug fixes and TODO cleanup.

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
PTAL. Just some minor cleanup.

### CODE REVIEW

File: src/local/butler/reproduce.py [hunk 1]
```diff
@@ -272,7 +279,15 @@ def _reproduce_crash(testcase_id, build_directory):
...
   """Attempt to reproduce a crash then report on the result."""
-  result = _reproduce_crash(args.testcase, args.build_dir)
+  try:
+    result = _reproduce_crash(args.testcase, args.build_dir)
+  except ReproduceToolException as exception:
+    print(exception)
+    return
+
+  if result.is_crash():
+    print('Test case reproduced successfully:\n')
```
Comment (inferno-chromium) [inline src/local/butler/reproduce.py]: Remove :\n from this and Output:\n from next

just do a common
print('Output:\n' + result.get_stacktrace())

File: src/local/butler/reproduce.py [hunk 2]
```diff
@@ -272,7 +279,15 @@ def _reproduce_crash(testcase_id, build_directory):
 
 def execute(args):
   """Attempt to reproduce a crash then report on the result."""
-  result = _reproduce_crash(args.testcase, args.build_dir)
+  try:
+    result = _reproduce_crash(args.testcase, args.build_dir)
+  except ReproduceToolException as exception:
+    print(exception)
```
Comment (inferno-chromium) [inline src/local/butler/reproduce.py]: Why not just raise ?

Comment (mbarbella-chromium) [reply src/local/butler/reproduce.py]: It just seems like cleaner output. The cases this catches are supposed to be unrecoverable errors that we've handled explicitly.

File: src/local/butler/reproduce.py (hunk 1 above)
Comment (mbarbella-chromium) [reply src/local/butler/reproduce.py]: Done

**Pull request merged**